packages/ui/src/ValueViewComponent/index.tsx

```diff
@@ -11,21 +11,13 @@ import { useDensity } from '../hooks/useDensity';
 
 type Context = 'default' | 'table';
 
-const Row = styled.span<{ $context: Context; $priority: 'primary' | 'secondary' }>`
+const Row = styled.span`
   display: flex;
   gap: ${props => props.theme.spacing(2)};
   align-items: center;
-  width: min-content;
+  width: max-content;
   max-width: 100%;
   text-overflow: ellipsis;
-
-  ${props =>
-    props.$context === 'table' && props.$priority === 'secondary'
-      ? `
-        border-bottom: 2px dashed ${props.theme.color.other.tonalStroke};
-        padding-bottom: ${props.theme.spacing(2)};
-      `
-      : ''};
```

Comment:
When rows in the assets table contained equivalent values, they'd be taller because of this styling. So Sam and I decided we should only underline the text:
<img width="196" alt="image" src="https://github.com/user-attachments/assets/5e2a7dc5-0eba-4582-893c-fef04579bd43">

Reply:
Intuitively it feels like a clickable element. So, is the underline needed just to replace the padding?

Reply:
Hm, true. Maybe something to revisit with Sam 🤔 

re: the padding: there was already an underline beneath it. It just _also_ had padding, which made it a different size than a non-`secondary` version of the component, causing visual issues.

This change simply moves the underline from beneath the _entire_ component, to just beneath the text.